eos-update-flatpak-repos

```diff
@@ -1213,6 +1213,34 @@ FLATPAKS_TO_MIGRATE = [
         "new-origin": "flathub",
         "mangle-metadata": mangle_firefox_metadata,
     },
+    # migrate Hack apps from eos-apps to flathub (T31133)
+    {
+        "name": "com.hack_computer.Clubhouse",
+        "new-name": "com.hack_computer.Clubhouse",
```

Comment:
I don't _think_ that `new-name` is needed if the names are the same. and when I run this, I get:

```
WARNING:root:Avoiding undefined behaviour, cannot rename an app with extensions: com.hack_computer.Clubhouse
```

So I think you need:

```suggestion
```